src/blocks/pacman.rs

```diff
@@ -134,11 +136,7 @@ impl PacmanConfig {
             // most likely a mistake: {count}, {pacman}, {aur}, {both} not found in format string
             Err(ConfigurationError(
                 "pacman".to_string(),
-                (
-                    "No formatter ({count}|{pacman}|{aur}|{both}) found in format string"
-                        .to_string(),
-                    "invalid format".to_string(),
-                ),
+                "No formatter ({count}|{pacman}|{aur}|{both}) found in format string".to_string(),
```

Comment:
IIRC we don't do this kind of check for the other blocks, do we?

Reply:
No, we don't. By the way, from a closer look it doesn't even work with the new formatting.

Reply:
I'm fine with removing the check here.